admin/src/utils/ita.js

```diff
@@ -70,6 +73,8 @@ function getCategoryDescription(category, provider) {
       return "Replace colors in the app.";
     case "links":
       return "Replace links in the app.";
+    case "auth":
```

Comment:
I would have first thought to put this in email settings. Looks like you thought so too with the email settings hints. Seems find to do this for now but eventually we might want to rework admin panel so that storage location is not tied to where we are showing it in the UI.